### ISSUE TITLE
Bare blocks...

### ISSUE THREAD

**Issue by tlhackque** [perltidy/perltidy#140]
I ran into this behavior with bare blocks.  I don't think it helps to treat the `{{ ... }}` the same way as nested loops.   

From the programmer's point of view, in this case the `{{` and `}}` are each one token, and the extra indentation isn't helpful.

I didn't see bare blocks mentioned in the documentation, so if there's a magic option, I missed it.

Here's a test case (viscously pared down from a real script, with some names changed to protect the guilty).

The real script is actually useful; the exits are a list of buffer overflow checks parsing nested TLV.   If it was a subroutine, it would look like a list of early `return`s.  But here inline bare blocks are the natural expression.  Anyhow, here's the sample.

[bare.tar.gz](https://github.com/perltidy/perltidy/files/15210777/bare.tar.gz)

```sh
perltidy -v
This is perltidy, v20240202.03
```
```perl
PLACE:
{{
last PLACE if( $first );
$x = 7;
$y = 13;
}}

print( "lost\n" );
```
Which produces:
```perl
PLACE:
    {
        {
            last PLACE if( $first );
            $x = 7;
            $y = 13;
        }
    }

    print( "lost\n" );
```
I'd expect something more like:
```perl
PLACE:
    {{
        last PLACE if( $first );
        $x = 7;
        $y = 13;
    }}

    print( "lost\n" );-
```

**Comment by shancock9** [comment]
The option ``--weld-nested``, or ``-wn``, is available to make nested block braces
behave as a single brace:

```
      # perltidy -wn
      PLACE:
        { {
            last PLACE if ($first);
            $x = 7;
            $y = 13;
        } }
```

It works for any number of braces:

```
    # perltidy -wn
    { { { { {
        print "Hello World\n";
    } } } } }
```

**Comment by tlhackque** [comment]
Thanks, that's close enough.  I missed that option - I did search the documentation for 'bare', as well as for 'block', as well as scanning the whole list -  and didn't see it.  I gave up on searching for ` do ` too soon - too many spurious matches, though today I see persistence would have paid off.

Neither "weld" nor "nested" triggered a match; I don't think of the bare block as nested (though it is).    For me, the `{{` is a single token that turns the bare block into an implicit `do` loop, allowing `next` and `last`. 

I don't expect you to pander to my peculiar intuition, but that's how I missed it.

As always, thanks for putting me on the right track.  I'll close this as a case of "user blindness".
